src/pages/BlogPage.js

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axiosInstance from "../api/axios";
+import "./pageStyle/BlogPage.css";
+
+const BlogPage = () => {
+  const [displayBlogs, setDisplayBlogs] = useState([]);
+  const [page, setPage] = useState(1);
+  let navigate = useNavigate();
+
+  const getBlog = async () => {
+    try {
+      const { data } = await axiosInstance.get();
+      const allPost = data.getBlogPost.docs;
+      setDisplayBlogs((prevPost) => [...prevPost, ...allPost]);
+      setPage(page + 1);
```

Comment:
Why?

Reply:
this is to render the blogs from the backend

Reply:
Why call `setPage` after fetching?

And where do you use the `page` state variable?

Reply:
> Why call `setPage` after fetching?
> 
> And where do you use the `page` state variable?
 `page` state variable is used as an argument inside the useEffect dep. array.

Reply:
I have removed the page state totally, allowed the widows to handle the scroll.